dom.go

```diff
@@ -750,6 +751,10 @@ func (d document) DocumentURI() string {
 	return d.Get("documentURI").String()
 }
 
+func (d document) ExecCommand(name string, showDefaultUI bool, valueArgument string) bool {
+	return d.Call("execCommand", name, showDefaultUI, valueArgument).Bool()
+}
```

Comment:
`valueArgument` is documented at https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand as:

> **aValueArgument**
> For commands which require an input argument (such as insertImage, for which this is the URL of the image to insert), this is a DOMString providing that information. Specify null if no argument is needed.

So I don't think we can use `string` type for it, because it wouldn't be possible to specify null.

We can use either `*string`, or `interface{}` and document that it needs to be a `string` or `nil`.

@dominikh, do you have thoughts on this?